src/board_converter.py

```diff
@@ -0,0 +1,36 @@
+from board import Board
+from point import Point
+
+
+class BoardConverter:
+    @staticmethod
+    def to_string(board: Board) -> str:
+        result = ''
+
+        for line in board.get_cells():
+            for cell in line:
+                result += '1 ' if cell else '0 '
+
+            result = result.rstrip(' ')
```

Comment:
looks like a bug

Reply:
here we override the previous value of `result`

Reply:
But we want to do this!
```
rstrip() вернет копию строки str с удаленными символами chars в конце строки.
```